pkg/controllers/managementuser/rbac/handler_base_test.go

```diff
@@ -229,3 +229,187 @@ func Test_gatherRoles(t *testing.T) {
 		})
 	}
 }
+
+func TestCompareAndUpdateClusterRole(t *testing.T) {
+	numUpdatesCalled := 0
```

Comment:
I don't think the subtests can be parallel, but the main test definitely can be.
```suggestion
	t.Parallel()
	numUpdatesCalled := 0
```